src/server.c

```diff
@@ -6081,7 +6110,8 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "eventloop_duration_sum:%llu\r\n", server.duration_stats[EL_DURATION_TYPE_EL].sum,
             "eventloop_duration_cmd_sum:%llu\r\n", server.duration_stats[EL_DURATION_TYPE_CMD].sum,
             "instantaneous_eventloop_cycles_per_sec:%llu\r\n", getInstantaneousMetric(STATS_METRIC_EL_CYCLE),
-            "instantaneous_eventloop_duration_usec:%llu\r\n", getInstantaneousMetric(STATS_METRIC_EL_DURATION)));
+            "instantaneous_eventloop_duration_usec:%llu\r\n", getInstantaneousMetric(STATS_METRIC_EL_DURATION),
+            "cluster_incompatible_ops:%lld\r\n", server.stat_cluster_incompatible_ops));
```

Comment:
i think it's a good idea not to reflect that metric at all when not enabled (or in cluster mode)

Reply:
ok, we can not  reflect that metric in cluster mode

Reply:
btw, it always should be 0 in cluster mode, do you mean we don't show this metric in cluster mode?

Reply:
Yes. I think we should avoid exposing it in cluster mode and also when the config is disabled. Since 0 is also a valid value when enabled, I would rather not show it when disabled.

Reply:
you are concerned that not enabling this feature but exposing this metric, 0 could also be misleading, right?

Reply:
yes

Reply:
done.
we show this metric that only when cluster mode is enabled and this feature is enabled